iothub_client/tests/iothubclient_uploadtoblob_e2e/iothubclient_uploadtoblob_e2e.c

```diff
@@ -353,32 +355,34 @@ static void uploadToBlobGetData(IOTHUB_CLIENT_FILE_UPLOAD_RESULT result, unsigne
     (void)uploadToBlobGetDataEx(result, data, size, context);
 }
 
-static void e2e_uploadtoblob_multiblock_test(IOTHUB_CLIENT_TRANSPORT_PROVIDER protocol, bool useExMethod, bool causeAbort)
+static void e2e_uploadtoblob_multiblock_test(IOTHUB_CLIENT_TRANSPORT_PROVIDER protocol, bool useExMethod, bool causeAbort, bool noData)
 {
     IOTHUB_CLIENT_RESULT result;
     IOTHUB_PROVISIONED_DEVICE* deviceToUse = IoTHubAccount_GetSASDevice(g_iothubAcctInfo);
     ASSERT_IS_NOT_NULL(deviceToUse);
-
-    IOTHUB_CLIENT_HANDLE iotHubClientHandle = IoTHubClient_CreateFromConnectionString(deviceToUse->connectionString, protocol);
-    ASSERT_IS_NOT_NULL(iotHubClientHandle, "Could not invoke IoTHubClient_CreateFromConnectionString");
-
-    uploadToBlobCauseAbort = causeAbort;
-    g_uploadToBlobStatus = UPLOADTOBLOB_CALLBACK_PENDING;
-    uploadBlobNumber = 0;
-    if (useExMethod)
+    if (deviceToUse != NULL)
```

Comment:
Do we need the `if (deviceToUse)` check?  Right above there's an `ASSERT_IS_NOT_NULL(deviceToUse)` so if this is NULL the test should immediately exit.

Reply:
Yes, this is flagged by SDL. It helps keep the noise down.